tests/test_modeling_blenderbot.py

```diff
@@ -151,70 +161,114 @@ def test_generation_same_as_parlai_3B(self):
+            1384,
+            5186,
+            20,
+            21,
+            8,
+            17,
+            50,
+            241,
+            1789,
+            6,
+            6299,
+            6,
+            9,
+            2147,
+            5,
+            2,
+        ]
         self.assertListEqual(expected_tokens, generated_utterances)
 
     @torch.no_grad()
     @slow
     def test_3bsam_forward(self):
         input_ids = _long_tensor([[1384]])  # sam
-        ys = torch.tensor([[1, 49, 15, 286, 474, 10, 1384, 5186, 20, 21, 8, 17,
-                           50, 241, 1789, 6, 6299, 6, 9, 2147, 5, 2]], dtype=torch.long)
+        ys = torch.tensor(
+            [[1, 49, 15, 286, 474, 10, 1384, 5186, 20, 21, 8, 17, 50, 241, 1789, 6, 6299, 6, 9, 2147, 5, 2]],
+            dtype=torch.long,
+        )
         logits, *_ = self.model.forward(input_ids, decoder_input_ids=ys)
-        #import ipdb; ipdb.set_trace()
+        # import ipdb; ipdb.set_trace()
 
         parlai = load_parlai_3B()
-        assert self.model.encoder.embed_tokens.weight[3, 3] == parlai.encoder.embeddings.weight[3,3]
+        assert self.model.encoder.embed_tokens.weight[3, 3] == parlai.encoder.embeddings.weight[3, 3]
         assert self.model.decoder.embed_tokens.weight[3, 3] == parlai.decoder.embeddings.weight[3, 3]
 
-
         self.assertEqual(num_parameters(self.model.encoder), 53613568)
         self.assertEqual(num_parameters(self.model.decoder), num_parameters(parlai.decoder))
 
-        scores, preds, encoder_states = parlai.forward(input_ids, ys=ys[:,1:])
+        scores, preds, encoder_states = parlai.forward(input_ids, ys=ys[:, 1:])
         enc_out, enc_mask = encoder_states
         assert self.model.encoder_states.shape == enc_out.shape
-        assert_tensors_close(self.model.encoder_states[:,:,3], enc_out[:,:,3], atol=1e-3)
-        assert_tensors_close(logits[:,:-1], scores, atol=1e-4)
-        desired_logits = torch.tensor([ 0.8507, -3.2680, -1.7374, -4.4045,  4.7195, -0.7223, -0.0766],
-                                      device=torch_device)
-        assert_tensors_close(desired_logits, logits[0,0,3:10], atol=1e-4)
+        assert_tensors_close(self.model.encoder_states[:, :, 3], enc_out[:, :, 3], atol=1e-3)
+        assert_tensors_close(logits[:, :-1], scores, atol=1e-4)
+        desired_logits = torch.tensor(
+            [0.8507, -3.2680, -1.7374, -4.4045, 4.7195, -0.7223, -0.0766], device=torch_device
+        )
+        assert_tensors_close(desired_logits, logits[0, 0, 3:10], atol=1e-4)
+
+
 
-from .test_modeling_bart import _long_tensor, assert_tensors_close
 
 class Blenderbot90MIntegrationTests(AbstractBlenderBotIntegrationTests):
     checkpoint_name = "sshleifer/blenderbot-90M"
     tokenizer_cls = BlenderbotSmallTokenizer
 
-
     @slow
     def test_tokenization_same_as_parlai(self):
         tok = self.tokenizer
-        self.assertListEqual(tok('sam'), [1384])
+        self.assertListEqual(tok("sam"), [1384])
```

Comment:
Which tokenizer is not working with you? All the tests for the 90M model tokenizer passed for me

Reply:
`tok("sam")` is a dict you need to use `tok("sam")['input_ids']`